### DISCUSSION TITLE
Reporting: Malicious DM response to issue post

### DISCUSSION THREAD

**Discussion by buzuddha** [community/community#142559] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hi all, 

Not sure where else to post this or how to contact GH about it:

I posted a [feature request to a github repo](https://github.com/Tishacy/SciDownl/issues/31) and received a private message to my personal email through github containing a link with a malicious piece of code. The response doesn't appear in the discussion thread. 

I opened on a [remote machine](https://www.browserling.com/), appears to be a captcha, but the "I am human button copies a powershell command and the captcha "test" is to hit Win+R, Ctrl+V, Enter.

I'm not sure where to report this, but wanted folks to be aware.  

Response is below:

_to fix your trouble check this solution click 
maybe this will solve your problem._

"click" is the link is below but I've broken it up with a space few spaces after the .nz 

ATTENTION THE COMPLETED LINK IS MALICIOUS
"https://www.calvary whangarei.co.nz /capt ha/index.html"

**Comment by samus-aran** [comment]
Hi @buzuddha , 👋🏻 We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our [abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam). Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.

Thank you!

**Comment by MasterInQuestion** [comment]
It should be your email somehow leaked somewhere, and phisher correlated your GitHub activity:
    Forged an email to you. (the sender can be forged; though wouldn't pass proper verification)

    Mostly nothing GitHub can help.
    Report to your email service provider, maybe.

**Comment by jablka** [comment]
I'm having the same problem now!, -comments with malicious links, appearing under my issues.
https://github.com/terkelg/awesome-creative-coding/issues/193
https://github.com/terkelg/awesome-creative-coding/issues/194

Unfortunately, I don't see option for "reporting a comment". 
So I'm reporting the user as such.

**Reply by jablka** [reply]
UPDATE, the comments have been already removed by GitHub staff.
